examples/offline/d4rl_cql.py

```diff
@@ -22,9 +22,9 @@
 
 def get_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--task", type=str, default="HalfCheetah-v2")
+    parser.add_argument("--task", type=str, default="Hopper-v2")
```

Comment:
why this change?

Reply:
Hopper is more challenging than HalfCheetah for offline RL in my experience